src/views/data/data-view.js

```diff
@@ -45,12 +45,7 @@ export class DataView extends LitElement {
   @property({ type: String }) multiSearchString;
   @property({ type: String }) sortMethod = 'position';
   @property({ type: String }) viewMode;
-  @property({ type: Array }) multiLingualMode = [
-    LANGUAGE_CODES.PALI,
-    LANGUAGE_CODES.SANSKRIT,
-    LANGUAGE_CODES.TIBETAN,
-    LANGUAGE_CODES.CHINESE,
-  ];
```

Comment:
We need this, otherwise the multilingual mode will fail when called externally (e.g. when calling the URL http://localhost:8080/skt/multilang/SA20sthtvbhu directly). This is due to a race condition - when the multilingual mode is called upon directly we cannot know which languages are available, this is why we assume that *all* are available. The return data from the backend will then limit the data to what is really available.

Reply:
When I copy and paste that URL it changes to: http://localhost:8080/skt/undefined/SA20sthtvbhu and asks me to select the view-mode. (also when this statement is in as above).
But I see what you mean ... just a bit annoying that every time I make a change the whole thing renders 3 times.

Reply:
I'll play around with it a bit better later ... got to go now.